src/index.rst

```diff
@@ -32,64 +32,73 @@ The main drivers of the Axom project are to:
   *  Provide capabilities for LLNL research codes, proxy apps, etc. that simplify technology
      transfer from research efforts into production applications
 
-.. toctree::
-   :maxdepth: 1
-   :caption: Quickstart Guide
-
-   docs/sphinx/quickstart_guide/index
-
-The Axom Quickstart Guide contains information about accessing the code,
-configuring and building, linking with an application, etc.
-
-================================
-Axom Software Documentation
-================================
-
-The following lists contain links to user guides and source code documentation
-for Axom software components:
-
-.. toctree::
-   :maxdepth: 1
-   :caption: Component User Guides
-
-   Slic (Simple Logging Interface Code) <axom/slic/docs/sphinx/index>
-   Lumberjack (Scalable parallel message logging and filtering) <axom/lumberjack/docs/sphinx/index>
-   Sidre (Simulation data repository) <axom/sidre/docs/sphinx/index>
-   Slam (Set-theoretic lightweight API for meshes) <axom/slam/docs/sphinx/index>
-   Spin (Spatial indexes) <axom/spin/docs/sphinx/index>
-   Quest (Querying on surface tool) <axom/quest/docs/sphinx/index>
-   Mint (Mesh data model) <axom/mint/docs/sphinx/index>
-   Primal (Computational geometry primitives) <axom/primal/docs/sphinx/index>
-   Inlet (Input decks) <axom/inlet/docs/sphinx/index>
-
---------------------------
-Source Code Documentation
---------------------------
-
-  *  `Axom <doxygen/html/index.html>`_
-  *  `Core <doxygen/html/coretop.html>`_
-  *  `Inlet <doxygen/html/inlettop.html>`_
-  *  `Lumberjack <doxygen/html/lumberjacktop.html>`_
-  *  `Mint <doxygen/html/minttop.html>`_
-  *  `Primal <doxygen/html/primaltop.html>`_
-  *  `Quest <doxygen/html/questtop.html>`_
-  *  `Sidre <doxygen/html/sidretop.html>`_
-  *  `Spin <doxygen/html/spintop.html>`_
-  *  `Slic <doxygen/html/slictop.html>`_
-  *  `Slam <doxygen/html/slamtop.html>`_
-
-Look for documentation to appear for new components as they are developed.
-
-Dependencies between modules are as follows:
-
-- Core has no dependencies, and the other modules depend on Core
+==========
+Components
+==========
+
+   * Inlet: Input deck parsing and information storage/retrieval
+   * Lumberjack: Scalable parallel message logging and filtering
+   * Mint: Mesh data model
+   * Primal: Computational geometry primitives
+   * Quest: Querying on surface tool
+   * Sidre: Simulation data repository
+   * Slam: Set-theoretic lightweight API for meshes
+   * Slic: Simple Logging Interface Code
+   * Spin: Spatial indexes
+
+=============
+Documentation
+=============
+
+  * :doc:`Quickstart Guide <docs/sphinx/quickstart_guide/index>`
+  *  `Source documentation <doxygen/html/index.html>`_
+
+.. list-table::
+   :align: center
+
```

Comment:
Should `core` be here too?
It has doxygen if not sphinx docs.

Reply:
I think I looked yesterday and found no sphinx docs for core.

Reply:
I'm willing to leave the link to `core` for later.  They're not quite right in the overall TOC.  I've agreed to write some sphinx docs (ATK-1163), and I propose to fix the doxygen docs and add them here at the same time.

Reply:
Once again, thank you @agcapps ! 🥇